modules/wizard/view-wizard-step-home-page.php

```diff
@@ -88,15 +97,15 @@
 </table>
 <?php endif; ?>
 <table id="untranslated-languages" class="striped">
-	<?php if ( ! is_null( $default_language ) ) : ?>
+	<?php if ( ! empty( $default_language ) ) : ?>
 		<caption><span class="icon-default-lang"></span> <?php esc_html_e( 'Default language', 'polylang' ); ?></caption>
 	<?php endif; ?>
 	<thead>
 		<?php if ( count( $untranslated_languages ) >= 1 ) : ?>
 			<tr>
 				<th><?php esc_html_e( 'We are going to prepare this page in', 'polylang' ); ?></th>
 			</tr>
-		<?php elseif ( false !== $home_page_language && count( $untranslated_languages ) <= 0 ) : ?>
+		<?php elseif ( ! empty( $home_page_language ) ) : ?>
```

Comment:
The code is changing here. Why?

Reply:
The second condition is useless.
As `PLL_Translatable_Object::get_language()` returns `PLL_Language` or `false` do you prefer to keep the first condition as before?